DataManagementSystem/Client/DataManager.py

```diff
@@ -1422,9 +1396,7 @@ def __removePhysicalReplica( self, storageElementName, lfnsToRemove, replicaDict
       oDataOperation.setValueByKey( 'TransferOK', 0 )
       oDataOperation.setValueByKey( 'FinalStatus', 'Failed' )
       gDataStoreClient.addRegister( oDataOperation )
-      errStr = "Failed to remove replicas."
-      log.debug( errStr, res['Message'] )
-      return res
+      log.debug( "Failed to remove replicas.", res['Message'] )
```

Comment:
because at low level, logging should be done by caller. this was just for debugging